### PR TITLE
fixes aloe cream splitting

### PR DESCRIPTION
**Pull request by SirDragooon**
## Fixes Aloe Cream Splitting (#8807) 

:cl:
- fix: fixes Aloe Cream splitting, no longer turns into Ointment after splitting

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml [hunk 1]
```diff
@@ -109,6 +109,12 @@
     netsync: false
     sprite: Objects/Specific/Hydroponics/aloe.rsi
     state: cream
+  - type: Item
+    HeldPrefix: Aloe Cream
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:113]: this held prefix probably doesn't work, just add the stack stuff imo

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml (hunk 1 above)
Comment (SirDragooon) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:113]: yeah wasn't sure about that aswell

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml [hunk 2]
```diff
@@ -109,6 +109,10 @@
     netsync: false
     sprite: Objects/Specific/Hydroponics/aloe.rsi
     state: cream
+  - type: Stack
+    stackType: AloeCream
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:113]: somehow missed this, this stack type doesn't actually exist and needs to get added in `Resources/Prototypes/Stacks/medical_stacks.yml`

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml (hunk 2 above)
Comment (SirDragooon) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:113]: alright fixed it 😅

Comment (SirDragooon) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:113]: hopefully

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**